### DISCUSSION TITLE
Deleting Session Data

### DISCUSSION THREAD

**Discussion by ScottVaden** [ChurchApps/ChurchAppsSupport#375] [Chums]
I am new to CHUMS and I was trying to set up classes and sessions.  After testing, I deleted the associated sessions and groups, but it appears they are still in the database.  Is there a way to delete them?

**Comment by tbyrd77** [comment]
Are you still having this problem?

**Reply by ScottVaden** [reply]
Yes, this is still an issue.

**Reply by tbyrd77** [reply]
Ok, I'll work on it today and see what I can come up with. Thanks

**Comment by ScottVaden** [comment]
Hey Terry,

Yes

Thanks for responding.  I did respond in the forum as well.

I also still have an issue with importing files and I am kind of stuck.

I have manually added about 75 names to the database, but now I have about
185 additional names that I want to import.

I have followed the procedure down through step 5, but if I perform step 6
as it is written, I will be introducing data in the database from the
sample file, so I don't want to do that without further guidance.  Either
step 6 is missing some instructions or something will happen on the back
end that I am not aware of.

This problem is more of a concern for me at this moment than the ticket I
had entered.

I just need to know how to proceed with this and any other future issue
that I may come across.

Thanks
Scott

**Comment by ScottVaden** [comment]
More on the importing issue...

It appears that If I follow the instructions as written, I will be
importing sample data into the database.

It is true I want to add 185 names for people, but I don't want to add
households, campus, etc from the sample files into the database.

I think the steps are leaving out the need for me to remove all the
information from the other tables and not to include the png files when I
compress all the tables to get ready to import.  ?

I will be back home at 1pm CST if you want to discuss.

Thanks
Scott Vaden
615-202-7635



[image: image.png]

**Comment by tbyrd77** [comment]
Hi Scott,
Yes, you are correct. You must replace all the data in the sample file with your data.
 <https://support.churchapps.org/chums/import-csv.html#>Unzip sample file and open the file people.csv replace data in this file with your data. Do not change the top row and be sure to save your edited file as people.csv

Serving Him,
Terry Byrd
918-860-0053

**Comment by ScottVaden** [comment]
Hey Terry,

So the instructions say to replace all the data in the *people.csv* file
and replace it with your data, but it doesn't say anything about emptying
out the data in the other files. (Which still have sample data in them).

attendance.csv
donations.csv
groupmembers.csv
groups.csv
services.csv

Since they still have data in them and you compress them with the
people.csv file and then import it, you will load sample data into your
database.  Which I don't think anyone wants to do.

I went ahead and emptied the data in the other csv files and also deleted
the png images which were the pictures for the people in the people.csv
file.  After cleaning all the data from all the files, I was then able to
compress it and import it into the database and everything appears to be
working correctly.

I just think you need to make that clear in the instructions.  I suspect if
others have imported people data into their database based on the
instructions, they have data in their database they don't want or need.

Scott

**Comment by ScottVaden** [comment]
Hey Terry,

I was just wanting to check back with you and make sure what I was saying
makes sense.

Thanks
Scott

**Comment by tbyrd77** [comment]
Hi Scott,
Yes, I see what you are saying. I'm surprised we haven't come upon this until now. It must be because most people download from Breeze or Planning Center. Did your download work? I'll go in and update the tutorial. Thanks for finding this for us! I'm still working on duplicating the problem with classes and sessions. I'll get back with you on that! Have a blessed day!

**Comment by ScottVaden** [comment]
Hi Terry,

I am not sure I understand the question you are asking: "Did the download
work?" If you are referring to the download of the sample file in the
tutorial (under "import from CSV")  then, yes.  Everything about the
tutorial worked with the exception of cleaning out all the data from the
other tables and removing the PNG files before compressing. (I followed the
apple instructions).

I saw there was a tutorial for importing from Breeze (didn't see one for
Planning Center).  I am not familiar with either of those software
packages, but I can only assume people in churches are using them and then
moving to ChurchApps because of the cost.

This church is not currently using anything for a master database of
people.  They do use Subsplash for their donations and media. They are
using something for their Kids ministry, but I haven't looked at that yet.
 They are also using Mailchimp for some newsletters.  So I have data from a
number of places that I am trying to pull data together in CSV format to
create a master database.  If I can find a package that will fulfill all
their needs that would be great.

I like what I see with ChurchApps so far, but support is pretty important.
I just need someone I can work with when I run into issues.  I know
ChurchApps works off donations which is great and gives a church a chance
to build before creating new expenses which is what they need right now for
sure.  So I am not expecting too much in the way of support, because I am
sure you guys are on a tight budget as well.

Thanks for your help and response.

Scott

**Comment by tbyrd77** [comment]
Hi Scott,
I looked at your account. Which Classes are you trying to delete?

**Comment by ScottVaden** [comment]
I think I was finally able to delete all the things concerning sessions and
pretty much everything except people and groups.  I do have this info in
attendance.

[image: image.png]

Thanks
Scott
